appveyor.yml

```diff
@@ -86,3 +82,11 @@
     name: NuGet public feed
     on:
       branch: master
+  - provider: GitHub    
+    release: $(GitVersion_NuGetVersion)
+    description: 'Release v$(GitVersion_NuGetVersion)'
+    auth_token:
+      secure: 2ijF4OfPYfIUTunubL/Z04ogADL9sx/gKW1jMRNJnAa1OpHZpSvCKmP0gvNs8C7h
```

Comment:
yes, correct. It is the `repo_public` access token for the `radical-bot` user encrypted by `AppVeyor`.